apps/email/js/cards/fld/account_item.html

```diff
@@ -1,4 +1,4 @@
 <a class="fld-account-item">
   <span class="selected-indicator"></span>
-  <span dir="auto" class="fld-account-name"></span>
-</a>
\ No newline at end of file
+  <span class="fld-account-name"></span>
```

Comment:
OK, if we go with this route for this section, then it seems like these other places listed below where we show the email address will need to be changed to match. Otherwise, we are likely to get bug reports of a mismatch in style between how the account is shown here (as the first line in fiddle shows) vs these other places (where the layout of the second line in the fiddle is used).

I am fine with whatever approach you think makes sense, as long as we can make a best effort for the display to be uniform across these different areas.

Elements with dir="auto", text-overflow:ellipsis:

* js/cards/tng/account_item.html: .tng-account-item-label. Tap on gear icon to show the "Settings" screen. This is used to show each account name.
* js/cards/settings_account.html: .tng-account-header-label. From "Settings" screen, tap an account to see Account Settings. This is the header.
* js/cards/settings_account_credentials.html: .tng-account-header-label.
* js/cards/settings_account_servers.html: .tng-account-header-label (BUG: missing dir="auto")
* js/cards/setup_account_password.html: .sup-email-display

For the last three, in Account Settings, scroll down to "Server settings" section and tap items labeled "Username", and "IMAP settings", or "SMTP settings" or "ActiveSync settings".

No specific styles on the element, but the parent element has dir="auto", text-overflow:ellipsis, which affects the child:

* js/cards/msg/peep_bubble.html: .msg-peep-content, parent: .msg-peep-bubble. When looking at an email with its body in the Message Reader, the bubble used for the email addresses in the header.
* js/cards/cmp/peep_bubble.html: .cmp-peep-name and .cmp-peep-address could contain account. Parent: .cmp-peep-bubble. When composing a message in Compose view, the bubble shown for email addresses in the header.
* js/marquee.js: . marquee-h-text: parent, marquee-h-wrapper. SPECIAL: the marquee-h-text sets overflow: visible so that we can scroll the whole address via JS. Tap on a bubble in Compose or Message Reader to inspect details of the name, provides actions that can be done with the email address.